Pod/Classes/Logo/UIView+Logo.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIView+Logo.swift
+//  Pods
+//
+//  Created by Derek Ostrander on 7/18/16.
+//
+//
+
+import UIKit
+
+public extension UIView {
```

Comment:
I'd say that's an implementation detail. It could just as easily return a Raze view or something sourced from PaintCode.